mir-module/src/main/resources/xsl/response-mir-utils.xsl

```diff
@@ -26,7 +26,7 @@
         </xsl:variable>
         <div class="col-12 col-md-5 text-left">
           <a tabindex="0" class="previous" href="{$link}" data-pagination=".caption:mods.title.main">
-            <span class="fas fa-chevron-left icon" />
+            <span class="fa fa-chevron-left icon" />
```

Comment:
fa is not longer supported by current font awesome library version 5
chevron left is part of the solid font lib (fas) now
So why do you like to redo this font awesome migration?